### PR TITLE
fix: made all our shaders srp batcher compatible

### PR DESCRIPTION
**Pull request by BrianAmadori**
This should save a lot of draw calls.

Changes:
- All shaders are SRP batcher compatible (all their properties inside the UnityPerMaterial constant buffer)
- Remove `MaterialPropertyBlocks` from `MaterialTransitionController`. We should avoid them now as they aren't compatible with the SRP batcher.

- Add a postprocess to all materials to optimize batching with the following rules:
    - All materials have _EMISSION and _NORMALMAP and a few additional keywords
    - renderQueue is sorted so all the materials with the same keywords are rendered together.
    - Triplanar and hologram material is rendered in a special queue, so it doesn't break batches with other geometry
    - Skinned mesh renderers are rendered in a special queue.
    - Different _Cull properties are grouped inside a special queue.

### CODE REVIEW

**Comment by BrianAmadori** [comment]
Just tested in min specs, around 0,0:ORG, chrome, ubuntu:
Master -> 20-25 fps
This PR -> 25-30 fps

**Review by AjimenezDCL — APPROVED** [review]

**Comment by BrianAmadori** [comment]
Sorry, the ~30 ish improvement was on medium quality. On basic quality around that same zone the FPS go up to 40.

**Review by pbosio — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MaterialTransitionController/MaterialTransitionController.cs [hunk 1]
```diff
@@ -48,7 +50,7 @@ Renderer targetRenderer
 
     public GameObject placeholder { get; private set; }
     public Renderer placeholderRenderer { get; private set; }
-    static MaterialPropertyBlock placeholderPropertyBlock;
+    //static MaterialPropertyBlock placeholderPropertyBlock;
```
Comment (AjimenezDCL) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MaterialTransitionController/MaterialTransitionController.cs]: Remove all the comments

**Review by AjimenezDCL — APPROVED** [review]

**Review by pbosio — APPROVED** [review]

**Review by pravusjif — CHANGES_REQUESTED** [review]
we still got problems with transparency
https://play.decentraland.zone/branch/fix/lit-shader-not-srp-compatible-issue/index.html?ENV=zone&position=55%2C-71&realm=odin-amber

**Review by pravusjif — APPROVED** [review]

**Pull request merged**